### PR TITLE
Adding values for liveness & readiness probe thresholds

### PR DESCRIPTION
**Pull request by adamcharnock**
This is an addition to the existing ``postgresql`` chart.

I have added the following values:

```
probes.liveness.initialDelay
probes.liveness.timeoutSeconds
probes.liveness.failureThreshold
probes.readiness.initialDelay
probes.readiness.timeoutSeconds
probes.readiness.failureThreshold
```

Each use the defaults already present.

## Reason

I'm adding these because the default ``probes.liveness.initialDelay`` value of 60 seconds was – for my cluster – not long enough for the initial database setup to take place. The pod would therefore get killed part way though setup, and before any login was created. The pod would then restart, believe the setup had been completed, and leave the database with no way to login.

I included the rest of the options for completeness.

### CODE REVIEW

**Review by timstoop — CHANGES_REQUESTED** [review]
As charts are immutable, you need to bump the version in Chart.yaml as well. The rest looks fine to me.

**Comment by adamcharnock** [comment]
Thanks @timstoop, I've increased the chart version from `0.8.4` to `0.8.5`

**Comment by adamcharnock** [comment]
I realised that the email address I gave to the Linux Foundation did not match my github email. I've fixed this now, hopefully that'll allow this to move forward soon.

File: stable/postgresql/templates/deployment.yaml [hunk 1]
```diff
@@ -57,18 +57,18 @@ spec:
             - sh
             - -c
             - exec pg_isready --host $POD_IP
-          initialDelaySeconds: 60
-          timeoutSeconds: 5
-          failureThreshold: 6
+          initialDelaySeconds: {{ default "60" .Values.probes.liveness.initialDelay | quote }}
```
Comment (unguiculus) [inline stable/postgresql/templates/deployment.yaml]: Remove the `default` function calls. They are redundant.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
Ping for CLA check.

**Comment by adamcharnock** [comment]
Ping for CLA check

**Review by unguiculus — APPROVED** [review]

**Comment by unguiculus** [comment]
@adamcharnock Are you still interested in the PR? If so, please merge in the current master, so we can proceed. Otherwise, please close it. Thanks.

**Comment by adamcharnock** [comment]
@unguiculus Done

**Comment by adamcharnock** [comment]
@unguiculus Merged again. I've also bumped the version number again.

If we cannot get this merged this time around then I'm probably going to back away from this PR, the level of procedural faff is starting to feel prohibitive for me.

**Comment by adamcharnock** [comment]
Thank you @unguiculus!

**Comment by ib-steffen** [comment]
seems like this change broke installation:

```
helm install --name postgres stable/postgresql

Error: release postgres failed: Deployment in version "v1beta1" cannot be handled as a Deployment: v1beta1.Deployment: Spec: v1beta1.DeploymentSpec: Template: v1.PodTemplateSpec: Spec: v1.PodSpec: Containers: []v1.Container: v1.Container: LivenessProbe: v1.Probe: FailureThreshold: readUint32: unexpected character: �, parsing 966 ...reshold":"... at {"apiVersion":"extensions/v1beta1","kind":"Deployment","metadata":{"labels":{"app":"postgresql","chart":"postgresql-0.14.1","heritage":"Tiller","release":"postgres"},"name":"postgres-postgresql","namespace":"default"},"spec":{"selector":{"matchLabels":{"app":"postgresql","release":"postgres"}},"template":{"metadata":{"labels":{"app":"postgresql","release":"postgres"}},"spec":{"containers":[{"args":null,"env":[{"name":"POSTGRES_USER","value":"postgres"},{"name":"PGUSER","value":"postgres"},{"name":"POSTGRES_DB","value":""},{"name":"POSTGRES_INITDB_ARGS","value":""},{"name":"PGDATA","value":"/var/lib/postgresql/data/pgdata"},{"name":"POSTGRES_PASSWORD","valueFrom":{"secretKeyRef":{"key":"postgres-password","name":"postgres-postgresql"}}},{"name":"POD_IP","valueFrom":{"fieldRef":{"fieldPath":"status.podIP"}}}],"image":"postgres:9.6.2","imagePullPolicy":"","livenessProbe":{"exec":{"command":["sh","-c","exec pg_isready --host $POD_IP"]},"failureThreshold":"6","initialDelaySeconds":"60","timeoutSeconds":"5"},"name":"postgres-postgresql","ports":[{"containerPort":5432,"name":"postgresql"}],"readinessProbe":{"exec":{"command":["sh","-c","exec pg_isready --host $POD_IP"]},"initialDelaySeconds":"5","periodSeconds":null,"timeoutSeconds":"3"},"resources":{"requests":{"cpu":"100m","memory":"256Mi"}},"volumeMounts":[{"mountPath":"/var/lib/postgresql/data/pgdata","name":"data","subPath":"postgresql-db"}]}],"volumes":[{"name":"data","persistentVolumeClaim":{"claimName":"postgres-postgresql"}}]}}}}

```

**Comment by desaintmartin** [comment]
Those values don't need to be quoted! Kubernetes wants integer, not string for at least failureThreshold.

**Pull request merged**